src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private ColorSupplier color = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(color.getRandomColor(), random.nextInt(MAX_NUMBER));

```

Comment:
Please, pay attention that in that case the maximum number would be 99, but the task is to get 100.
I would recommend do not increase MAX_NUMBER, but add 1 to the `nextInt` method call result.